src/tir/transforms/inject_ptx_async_copy.cc

```diff
@@ -79,7 +80,7 @@ class PTXAsyncCopyInjector : public StmtMutator {
         if (indices_lanes == 1) {
           auto src_offset = load->indices[0];
           auto dst_offset = store->indices[0];
-          Array<PrimExpr> args = {store->buffer->data, tir::Mul(dst_offset, PrimExpr(index_factor)),
+          Array<PrimExpr> args = {store->buffer->data, mul(dst_offset, PrimExpr(index_factor)),
```

Comment:
Since your description mentions this as a separate bug, can it either be split out into a separate PR, or (since it is a relatively small change), have a test case added for it?

Reply:
I open a new PR [here](https://github.com/apache/tvm/pull/16000) . Please have a check. 
I will open a new PR to fix dtype mismatch bug in PASS  `InjectPTXAsyncCopy` after symbolic strides PR is merged